articles/storage/files/storage-files-identity-ad-ds-enable.md

```diff
@@ -20,6 +20,9 @@ This article describes the process for enabling Active Directory Domain Services
 
 To enable AD DS authentication over SMB for Azure file shares, you need to register your Azure storage account with your on-premises AD DS and then set the required domain properties on the storage account. To register your storage account with AD DS, you create a computer account (or service logon account) representing it in your AD DS. Think of this process as if it were like creating an account representing an on-premises Windows file server in your AD DS. When the feature is enabled on the storage account, it applies to all new and existing file shares in the account.
 
+> [!NOTE]
+> This article uses Azure Files native FQDNs (\<your-storage-account\>.file.core.windows.net) for file share access. It is also possible to use a CNAME record to provide access with your own domains. See [Use Azure Files with multiple Active Directory (AD) forests](storage-files-identity-multiple-forests.md) for details.
```

Comment:
```suggestion
> This article uses Azure Files native FQDNs (\<your-storage-account\>.file.core.windows.net) for file share access. It is also possible to use a CNAME record to provide access with your own domains. See [Use Azure Files with multiple Active Directory forests](storage-files-identity-multiple-forests.md) for details.
```
Deleting acronym per [style guidance](https://styleguides.azurewebsites.net/Styleguide/Read?id=2719&topicid=31075) for this terminology.